src/models/imposter.js

```diff
@@ -54,6 +54,9 @@ function create (Protocol, request) {
 
                 result.requests = server.requests;
                 result.stubs = server.stubs();
+                result.stubs.forEach(function (stub) {
```

Comment:
This feels off to me. Why not just `result.state = server.state`? Is there a reason we're showing the state with each stub?